dev-python/ipython/ipython-4.0.0-r1.ebuild

```diff
@@ -63,7 +62,8 @@ DEPEND="${CDEPEND}
 		dev-python/rpy[${PYTHON_USEDEP}]
 		>=dev-python/sphinx-1.1[${PYTHON_USEDEP}]
 		>=www-servers/tornado-4.0[${PYTHON_USEDEP}]
-	)"
+	)
+	wxwidgets? ( dev-python/wxpython[${PYTHON_USEDEP}] )"
```

Comment:
Please move to CDEPEND and modify according to in https://github.com/gentoo/gentoo/blob/master/dev-python/ipython/ipython-3.2.1.ebuild#L34